build.sbt

```diff
@@ -43,6 +46,32 @@ lazy val runtime1         = Project("daffodil-runtime1", file("daffodil-runtime1
                               .dependsOn(io, lib % "test->test", udf, macroLib % "compile-internal, test-internal")
                               .settings(commonSettings, usesMacros)
 
+val runtime2StaticLib     = Library("libruntime2.a")
+lazy val runtime2         = Project("daffodil-runtime2", file("daffodil-runtime2")).configs(IntegrationTest)
```

Comment:
Agreed with your suggestions. Though, I suggest we do the rename separately (except maybe runtime2 since it's not merged yet), if we do decide to change names. Sometimes github doesn't detect that a change is a rename and it makes it really difficult to see what changes are just renames and what are actual changes. So mixing changes with renames can sometimes make it hard to review. I too would like to hear what other devs think of this. It might also be worth a discussion on the dev list, since not everyone is paying detailed attention to this review. This comment is really unlreated to the PR, but just somethign I thought was worth bringing up since we now have an actual second runtime.